docker/run.sh

```diff
@@ -15,6 +15,15 @@ SHM_SIZE=8192m
 gpu_arg=""
 if $ATTACH_GPUS ; then gpu_arg="--gpus all" ; fi
 image=${1:-flexflow}
+cuda_version=${cuda_version:-11.1.1}
+
+if [[ "$cuda_version" != @(11.1.1|11.2.2|11.3.1|11.5.2|11.6.2|11.7.1|11.8.0) ]]; then
```

Comment:
could we also autodetect the cuda version in the `run.sh`? And could you add a message telling the user which cuda version the script is building for ? E.g. "Building FlexFlow docker image for CUDA 11.8"